lib/portlayer/exec/container.go

```diff
@@ -192,7 +190,6 @@ func (c *Container) Commit(ctx context.Context, sess *session.Session, h *Handle
 		}
 
 		c.State = *h.State
-		commitEvent = events.ContainerStopped
```

Comment:
This change introduced a "Creating" state -- so the need to update the handle state in commit was eliminated.  

The elimination of the stopped was a mistake -- THANKS!